lib/mina/infinum/ecs/params.rb

```diff
@@ -0,0 +1,66 @@
+require 'mina/infinum/ecs/aws'
+
+namespace :params do
+  desc <<~TXT
+    Print AWS Param Store params to stdout
+
+    Params are fetched from path "/:cluster/*".
+  TXT
+  task read: ['aws:profile:check'] do
+    ensure!(:cluster)
+
+    puts get_params.map(&:as_env)
+  end
+
+  desc <<~TXT
+    Save AWS Param Store params to .env file
+
+    Params are fetched from path "/:cluster/*".
```

Comment:
I tried it on Tyrenet, and this assumption isn't true on this project; the relationship between cluster and SSM Parameter Store is as follows:
| Cluster | SSM Param Store path |
|---|---|
| tyrenet-staging | /tyrenet/staging/app/zeus |
| tyrenet-uat | /tyrenet/uat/app/zeus |
| tyrenet-production | /tyrenet/production/app/zeus |

This is just a data point for you to decide on how you want to parametrize the `path`